ci/docs/build.sh

```diff
@@ -57,10 +57,9 @@ for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*
```

Comment:
Trailing whitespace is automatically removed by pre-commit hooks `trailing-whitespace` and `end-of-file-fixer`.